RevitLookup/Core/CollectorExtensions/DataFactory.cs

```diff
@@ -105,7 +105,8 @@ public Data Create(MethodInfo methodInfo)
 
         if (info.GetParameters().Length > 0 || info.ReturnType == typeof(void))
             return null;
-
+        if (declaringType == typeof(PrintManager) && info.Name == nameof(PrintManager.SubmitPrint))
+            return new BoolData("SubmitPrint",true);
```

Comment:
use it `return new BoolData(nameof(PrintManager.SubmitPrint), false);`

a space after the comma, and perhaps it's better to display false, the print job was not sent